From 56fa3b07ee7ac9d66315eb3d4e1caff81afd7588 Mon Sep 17 00:00:00 2001
From: Paul Keith <javelinanddart@gmail.com>
Date: Wed, 27 Jun 2018 01:00:49 +0200
Subject: [PATCH 13/14] Settings: Disable manual sim provisioning by default

* Since this menu is available for single sim devices too,
  make multi sim devices manually enable this overlay

Change-Id: Iac21072affe66fd5c54a1e48fd7685229369c9b4
---
 res/values/config.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/res/values/config.xml b/res/values/config.xml
index cc679a186f..3ea04161f7 100755
--- a/res/values/config.xml
+++ b/res/values/config.xml
@@ -126,5 +126,5 @@
 
     <!-- Does the device allow for manual subscription provisioning? Only works for multi-sim devices,
          and currently depends on QC's proprietary telephony stack -->
-    <bool name="config_enableManualSubProvisioning">true</bool>
+    <bool name="config_enableManualSubProvisioning">false</bool>
 </resources>
-- 
2.17.1

